gspread/worksheet.py

```diff
@@ -2095,8 +2095,8 @@ def add_dimension_group_columns(self, start, end):
 
         .. note::
 
-            API behavior with nested groups and non matching ``[start:end[``
-            range can be found here `Add Dimension Group Request`_
+            API behavior with nested groups and non-matching ``[start:end]``
+            range can be found here: `Add Dimension Group Request`_
```

Comment:
the open square-brackets at the end is a mathematical notation to mention that the end index is not included.

I learned at school to write it like this but it seems from what I find online the general way to write it would be: `[start:end)`
If you have time may be you could update it in your PR using round-brackets on each `end` occurence please, or I'll do later otherwise.